lib/media/segment_utils.js

```diff
@@ -314,6 +388,31 @@ shaka.media.SegmentUtils = class {
     }
     return ret;
   }
+
+  /**
+   * Prioritizes Dolby Vision if supported. This is necessary because with
+   * Dolby Vision we could have hvcC and dvcC boxes at the same time.
+   *
+   * @param {!Array.<string>} codecs
+   * @return {!Array.<string>} codecs
+   * @private
+   */
+  static chooseBetterCodecs_(codecs) {
```

Comment:
The implementation of this is due to https://chromium-review.googlesource.com/c/chromium/src/+/5058372